### DISCUSSION TITLE
what happened to mobx-react-lite?

### DISCUSSION THREAD

**Discussion by RussianDragon** [mobxjs/mobx#3755] [General]
Hi.
On last week I am downloaded npm package mobx-react-lite version 4
And it is not working. In Readme npm package mobx-react-lite not have information about 4  version.

Repository github package  mobx-react-lite - closed, but on site mobx recommend  use mobx-react-lite.
I don't understand, what happened to mobx-react-lite?
It package actual or other version  actual? 
What used if mobx-react-lite obsolete?

**Comment by mweststrate** [comment]
It's not obsolete, the repo is just acrchived as it was merged into the
main repo, as stated at the top of the readme

**Reply by RussianDragon** [reply]
Ok.
Maybe you know what new 4 version mobx-react-lite?
Because I have problems with rendering after update fields in version 4. If change version on 3 package mobx-react-lite , problem disappear.

**Reply by iChenLei** [reply]
@RussianDragon  you can check the [diff](https://github.com/mobxjs/mobx/compare/mobx-react-lite%403.4.3...mobx-react-lite%404.0.3) between v4.0.3 and v3.4.3 , and the [changelog](https://github.com/mobxjs/mobx/blob/main/packages/mobx-react-lite/CHANGELOG.md) of mobx-react-lite

**Comment by mweststrate** [comment]
For things that break, please follow the new issue template, "it doesn't
work" is not enough info to help with
